apps/wing/src/commands/test.ts

```diff
@@ -29,6 +30,16 @@ export interface TestOptions extends CompileOptions {
 }
 
 export async function test(entrypoints: string[], options: TestOptions): Promise<number> {
+  const expandedEntrypoints = await glob(entrypoints);
+
+  if (expandedEntrypoints.length === 0) {
+    // Check if it's the default entrypoint
+    if (entrypoints[0] === "*.test.w") {
+      throw new Error("No '.test.w' files found in current directory.");
+    }
+    throw new Error("No matching files found in current directory.");
+  }
```

Comment:
I think the error can be simplified just by spitting the requested globs back to the user

```suggestion
  if (expandedEntrypoints.length === 0) {
    throw new Error(`No matching files found in current directory. (${entrypoints.join(", ")})`);
  }
```